### DISCUSSION TITLE
Installation error

### DISCUSSION THREAD

**Discussion by manish050994** [Sunbird-cQube/community#851] [General]
Dear Team,

While doing the installation of release-v5.0.3 it it showing  cannot open config_files/config.yml (No such file or directory).

[PFA]
![Screenshot (28)](https://github.com/Sunbird-cQube/community/assets/113786506/af886c26-3f5f-48fe-9ce4-d80241ecf337)

**Comment by RNarmatha** [comment]
Hi @manish050994,

If you are attempting a reinstallation, kindly remove the current Docker images and proceed.

I attempted to reproduce this issue during the reinstallation, but the installation is functioning as expected . Please inform me of your results after you try it.

Thanks
